ui/lib/pki/addon/components/page/pki-tidy-status.hbs

```diff
@@ -0,0 +1,106 @@
+<Toolbar>
+  <ToolbarActions>
+    {{! TODO remove action here in favor of cancel action in status banner }}
+    <button
+      type="button"
+      class="toolbar-link"
+      {{on "click" (fn (mut this.confirmCancelTidy) true)}}
+      data-test-confirm-cancel-tidy
+    >
+      Cancel tidy
+      <Icon @name="chevron-right" />
+    </button>
+    <div class="toolbar-separator"></div>
+    <button
+      type="button"
+      class="toolbar-link"
+      {{on "click" (fn (mut this.tidyOptionsModal) true)}}
+      data-test-pki-tidy-options-modal
+    >
+      Tidy
+      <Icon @name="chevron-right" />
+    </button>
+    {{#if @autoTidyConfig.enabled}}
+      <ToolbarLink @route="tidy.auto" data-test-pki-auto-tidy-config>
+        Auto-tidy configuration
+      </ToolbarLink>
+    {{/if}}
+  </ToolbarActions>
+</Toolbar>
+
+{{! TIDY OPTIONS MODAL }}
+<Modal
+  @type="info"
+  @title="Tidy this mount"
```

Comment:
Since we didn't have explicit designs I went ahead and built out some of this modal, I followed the pattern used for the rotate root modal